src/main/java/gillycarpetaddons/mixins/FlowerBlock_replaceFlowersMixin.java

```diff
@@ -0,0 +1,38 @@
+package gillycarpetaddons.mixins;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.FlowerBlock;
+import net.minecraft.block.Material;
+import net.minecraft.item.ItemPlacementContext;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+
+import static gillycarpetaddons.GillyCarpetAddonsSettings.replaceableFlowers;
+
+@Mixin(FlowerBlock.class)
+public abstract class FlowerBlock_replaceFlowersMixin extends Block {
```

Comment:
Class and name should be `FlowerBlock_ReplaceFlowersMixin` to keep consistency.

Reply:
oh ffs one character change